fmlearlydisplay/src/main/java/net/minecraftforge/fml/earlydisplay/DisplayWindow.java

```diff
@@ -0,0 +1,531 @@
+package net.minecraftforge.fml.earlydisplay;
+
+import joptsimple.OptionParser;
+import net.minecraftforge.fml.loading.FMLConfig;
+import net.minecraftforge.fml.loading.FMLLoader;
+import net.minecraftforge.fml.loading.FMLPaths;
+import net.minecraftforge.fml.loading.ImmediateWindowProvider;
+import net.minecraftforge.fml.loading.progress.StartupNotificationManager;
+import org.jetbrains.annotations.Nullable;
+import org.lwjgl.PointerBuffer;
+import org.lwjgl.glfw.*;
+import org.lwjgl.opengl.GL;
+import org.lwjgl.opengl.GL30;
```

Comment:
Could pick any of these imports to use, but, to avoid errors I'd recommend using the GL classes with a C suffix (ie: GL32C) to ensure that no compatibility profile specific functions/constants are used.

Even more minor, but I'll also note the inconsistent style using static imports sometimes. a single `import static org.lwjgl.opengl.GL32C.*;` will get every OpenGL function and constant that can always be used.

Reply:
Tried to clean that up. thanks!